android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -712,9 +712,9 @@ public String getDeviceNameSync() {
   public String getSerialNumberSync() {
     try {
       if (Build.VERSION.SDK_INT >= 26) {
-        if (getReactApplicationContext().checkCallingOrSelfPermission(Manifest.permission.READ_PHONE_STATE) == PackageManager.PERMISSION_GRANTED) {
-          return Build.getSerial();
-        }
+        // There are a lot of conditions to access to getSerial api
+        // For details, see https://developer.android.com/reference/android/os/Build#getSerial()
+        return Build.getSerial();
```

Comment:
Just to make sure I checked to make sure SecurityException was truly a subclass of Exception so this will fall through and return "unknown", it is https://developer.android.com/reference/java/lang/SecurityException